pkgs/development/python-modules/jaxlib/bin.nix

```
@@ -113,7 +114,7 @@ buildPythonPackage rec {
     done
   '';
 
-  propagatedBuildInputs = [ absl-py flatbuffers scipy ];
+  propagatedBuildInputs = [ absl-py flatbuffers scipy ml-dtypes ];
```

nit: please alphabetize
```suggestion
  propagatedBuildInputs = [ absl-py flatbuffers ml-dtypes scipy ];
```

---

Done